### ISSUE TITLE
Page CaptureContent for network content body not getting set correctly

### ISSUE THREAD

**Issue by pharzan** [baudehlo/node-phantom-simple#122]
I'm trying to set the network [capture content ](https://docs.slimerjs.org/current/api/webpage.html#webpage-capturecontent) as I would using plain slimerJS with the following:

```
page.captureContent = [/html/, /text/];
```

using the getter setter functions with the node-slimerjs bridge using the following code:

```
page.set('captureContent',[/text/,/html/]);
```

but the setter function isn't setting the value correctly and I'm left with the empty body for the network responses with text/\* and html/\* types. 
After setting the captureContent if I try to read the value with console.log I can see that only an empty object has been set to the capture content:

```
page.get('captureContent', function(err,val){
        console.log('captureContent::: ',val)
        });
```

from node console:
_captureContent::: [ {}, {} ]_

I also tried creating new regex objects and setting them using the setter function but got the same result.

```
var re = new RegExp("html");
var re2 = new RegExp("text");
```

**Comment by awlayton** [comment]
I don't think node-phantom-simple supports passing regexes. I think the communication is all in JSON, so only things that JSON has will work.

**Comment by puzrin** [comment]
Looks like implementation restriction. Need to replace JSON data serializer to something better with date/regex support.

**Comment by awlayton** [comment]
Perhaps something like https://github.com/vkiryukhin/jsonfn?

**Comment by puzrin** [comment]
@awlayton yes. I also remember `JASON`, but it was not updated many years, and we dropped it from our project. I did not tracked current state of libs about this topic, but any well-supported will be ok.

PR will be accepted. Or of that's not so critical, that can be done by me sometime in far future (support from my side is not very active)

**Comment by awlayton** [comment]
I might be able to make a PR sometime in the future, but I am rather busy as of late.

**Comment by puzrin** [comment]
The same sutiation :) . We joined to do bulk changes/fixes for 1.0 -> 2.0, but those are complete and we are busy with another things. I can do quickly only critical fixes & release accepted PRs.

**Comment by crackjack** [comment]
any new development in this front?
